### DISCUSSION TITLE
Is there a way to print a statusline at the bottom of a kitty window?

### DISCUSSION THREAD

**Discussion by neelchakrabrty** [kovidgoyal/kitty#7817] [Q&A] [answered]
I am looking for something like [shox](https://github.com/liamg/shox). Are there any alternatives, or is there a native way to do it?

**Comment by kovidgoyal** [comment, marked as answer]
https://github.com/kovidgoyal/kitty/discussions/4447

**Reply by neelchakrabrty** [reply]
Thank you. It's amazing just how much kitty can do!
